videos/validate-gif-batch.py

```
@@ -1240,10 +1241,77 @@ def expect_truth_failure(
+        raise RuntimeError(
+            "repository truth self-test 'unreachable baseline without committed brief' "
+            "did not fail closed"
+        )
+    # Lock the other direction: a REACHABLE pin must stay authoritative, never
+    # silently replaced by the fallback. v1.2.1 is on main and predates the
+    # ghost-vitals fixture, so citing that fixture under the v1.2.1 pin must
+    # fail with "did not exist at baseline" naming the pinned commit. If a
+    # regression routed reachable pins through the fallback, the fixture WOULD
+    # exist at the brief's last commit and this validation would pass.
+    tag_commit = subprocess.run(
+        ["git", "rev-parse", "--verify", "v1.2.1^{commit}"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)